apps/asap-server/test/controllers/events.test.ts

```diff
@@ -648,6 +649,70 @@ describe('Event controller', () => {
     });
   });
 
+  describe('Event groups', () => {
+    const eventId = 'group-id-1';
+
+    test('Should return one group when event calendar is referenced by multiple groups', async () => {
```

Comment:
```suggestion
    test('Should return the first group when event calendar is referenced by multiple groups', async () => {
```
It may seem like a non significant detail but actually this test outlines the exact requirement that we got. It's then more obvious that when there is more than one group attached we take whichever one came back first from squidex (which means its completely random)